ddtrace/contrib/asgi/middleware.py

```diff
@@ -85,11 +85,15 @@ async def __call__(self, scope, receive, send):
         if scope["type"] != "http":
             return await self.app(scope, receive, send)
 
-        headers = _extract_headers(scope)
-
-        trace_utils.activate_distributed_headers(
-            self.tracer, int_config=self.integration_config, request_headers=headers
-        )
+        try:
+            headers = _extract_headers(scope)
+        except Exception:
+            log.warning("failed to decode headers", exc_info=True)
```

Comment:
should we make this more actionable?

e.g. `failed to decode headers, cannot parse distributed tracing headers for request` ?

Reply:
"actionable"

Reply:
:+1: yes! This log message sucked lol